src/shared/components/case-editor/services/case.notifier.ts

```diff
@@ -27,9 +29,22 @@ export class CaseNotifier {
         .pipe(
           map(caseView => {
             this.cachedCaseView = plainToClassFromExist(new CaseView(), caseView);
+            this.setBasicFields(this.cachedCaseView.tabs);
             this.announceCase(this.cachedCaseView);
             return this.cachedCaseView;
           }),
         );
     }
+    public setBasicFields(tabs: CaseTab[]): void {
+      tabs.forEach((t) => {
+        const caseName = t.fields.find(f => f.id === CaseNotifier.CASE_NAME);
+        const caseLocation = t.fields.find(f => f.id === CaseNotifier.CASE_LOCATION);
+        if (caseName || caseLocation) {
+          this.cachedCaseView.basicFields = {
```

Comment:
I don't think this will work. If the fields are separate tabs, the second tab will clear out the field on the first tab. 
Also it's slightly inefficient, in that it inspects all tabs. I would do something like this:
tabs.some(t => {
        const caseName = t.fields.find(f => f.id === CaseNotifier.CASE_NAME);
        const caseLocation = t.fields.find(f => f.id === CaseNotifier.CASE_LOCATION);
        if (caseName && caseLocation.value) {
          this.cachedCaseView.basicFields.caseNameHmctsInternal = caseName.value;
        }
        if (caseLocation && caseLocation.value) {
          this.cachedCaseView.basicFields.caseManagementLocation = caseLocation.value;
        }
        return this.cachedCaseView.baseFields.caseName && this.cachedCaseView.basicFields.caseManagementLocation;

       }